GoogleForCreators/web-stories-wp
# PPC 2.0: Thumbnail Edge Case Poster Image Backup

## Context

PPC 2.0 + [slack convo about backups if no poster image](https://xwp.slack.com/archives/C01ESBB8ZN1/p1624921376196400?thread_ts=1624477840.171600&cid=C01ESBB8ZN1)

## Summary

Show the first frame of the video as an image if there is no `poster` on the resource in the PPC

## Relevant Technical Choices

Changed this at the `LayerIcon` level, so this change will also go into effect in the layer panel since the `LayerIcon` is a shared component.

~If we only want this to be applied to the cards in the PPC just let me know and I can update that.~ Updated it so the ppc is the only thing that is changed.

## To-do

n/a

## User-facing changes

||Before|After|
|--|--|--|
|Checklist|![image](https://user-images.githubusercontent.com/22185279/123903933-d4635480-d92c-11eb-8ca8-0c62f0a1785f.png)|![image](https://user-images.githubusercontent.com/22185279/123904006-f3fa7d00-d92c-11eb-818e-83879546497b.png)|

## Testing Instructions

Easiest way would be to set `iconImage` in `assets/src/edit-story/elements/video/icon.js` to `false`. Should see it in action that way

Otherwise, the best way would be to upload a video and remove the poster image from the video resource.

### QA






## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #8070
